tests/kuttl/tests/dataplane-deploy-no-nodes-test/06-assert.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: edpm-compute-beta-nodeset
+  namespace: openstack
+spec:
+  services:
+  - download-cache
+  - bootstrap
+  env:
+  - name: ANSIBLE_FORCE_COLOR
+    value: "True"
+  nodes: {}
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+status:
+  conditions:
+  - message: NodeSet Ready
+    reason: Ready
+    status: "True"
+    type: Ready
+  - message: Deployment completed
+    reason: Ready
+    status: "True"
+    type: DeploymentReady
+  - message: Input data complete
+    reason: Ready
+    status: "True"
+    type: InputReady
+  - message: ServiceAccount created
+    reason: Ready
+    status: "True"
+    type: ServiceAccountReady
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: SetupReady
+  configHash: n696hcdh665h56ch74hc4h559hc5hdch5cdh5b8hb5h694hd8h65bh696h56chfhc4hcbh64fh5b8h5b5h9dh58h8bh98h97hf6h8h5cdhf9q
```

Comment:
Is there any point of asserting these config hashes? I had removed them from other tests earlier as someone adding anything has to run the tests to find these hashes or calculate them manually.

Reply:
The point is exactly that. If the hashes change it means something has changed in our code. If the change is expected there is nothing easier than filling the hash. If not, it indicates a potential issue.

This way the developer can't ignore necessity of revising existing tests.

Reply:
Hashes are not meant for code change, but the spec changes after deployment.  It's only meant to compared when specs are updated.

Reply:
I know hashes are computed from the data. You don't have to tell me that. But we are asserting on the result.
If, hypothetically, something changes in the way operator works with spec, or there is a change in kubebuilder defaults or one of many other things, the hash will change. Indicating that something is happening.

And if that happens in PR that shouldn't change the hash, we will know that it's touching something it shouldn't touch.